gammapy/image/lists.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+from numpy.testing import assert_allclose
+from astropy.units import Quantity
+from astropy.wcs import WCS
+from ..utils.testing import requires_dependency, requires_data
+from . import catalog
+from ..image import SkyMap
+from ..irf import EnergyDependentTablePSF
+from ..cube import SkyCube
```

Comment:
@OlgaVorokh - Can you please put this `SkyCube` import as a delayed import inside the `SkyImageList.to_cube` method?
Recently we discussed a Gammapy sub-package dependency structure, i.e. which sub-packages are lower-level and which are higher-level, to avoid circular imports. (I still have to write that up.)
`gammapy.image` is lower than `gammapy.cube`, i.e. should mostly avoid importing things from `gammapy.cube`, and where needed, do it as a delayed import.